### PR TITLE
media-video/obs-studio: version bump/removal

### PR DESCRIPTION
**Pull request by Chiitoo**
Just a 'cp' bump and a removal of old.

### CODE REVIEW

**Comment by Chiitoo** [comment]
A hotfix is already out, so do not merge this.

Thank you!

**Comment by Chiitoo** [comment]
Added a patch to fix the 'libdir' issue (also removing more old versions now).

**Comment by Chiitoo** [comment]
Broken at the moment, ugh.

**Comment by Chiitoo** [comment]
Nevermind, all better again!

File: media-video/obs-studio/obs-studio-23.2.1.ebuild [hunk 1]
```diff
@@ -68,6 +68,8 @@ DEPEND="
 "
 RDEPEND="${DEPEND}"
 
+PATCHES="$FILESDIR/$PN-23.2.1-use-correct-libdir.patch"
```
Comment (a17r) [inline media-video/obs-studio/obs-studio-23.2.1.ebuild]: `${FILESDIR}/${PN}`

Comment (Chiitoo) [reply media-video/obs-studio/obs-studio-23.2.1.ebuild]: Sigh.  I really can't read today.

**Pull request closed without merging**